CORE/elements/_core_element.rb

```diff
@@ -104,7 +104,7 @@ def visible?
 
   def present?
     begin
-      watir_element.wait_until_present(timeout:0)
+      watir_element.wait_until(timeout: 0, &:present?)
```

Comment:
why this change?

Reply:
Updating to the latest watir standard to remove a warning